kafka-connect/src/main/java/io/tabular/iceberg/connect/channel/Deduplicated.java

```diff
@@ -255,7 +255,7 @@ public String toString() {
           + eventGroupId
           + '\''
           + ", eventTimestamp="
-          + eventTimestamp
+          + eventTimestamp.toInstant().toEpochMilli()
```

Comment:
Easier to keep it as epoch milis in the log message.  Otherwise need to significantly rework DeduplicatedTests in an annoying fashion.  I think this is fine.